packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -0,0 +1,150 @@
+import designSystemDefaults, {
+    DesignSystem,
+    withDesignSystemDefaults,
+} from "../design-system";
+import {
+    ComponentStyles,
+    ComponentStyleSheet,
+    CSSRules,
+} from "@microsoft/fast-jss-manager";
+import {
+    applyLocalizedProperty,
+    Direction,
+    localizeSpacing,
+} from "@microsoft/fast-jss-utilities";
+import {
+    applyMixedColor,
+    ensureNormalContrast,
+    hoverContrast,
+    normalContrast,
+} from "../utilities/colors";
+import Chroma from "chroma-js";
+import { CarouselClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+function flipperStyles(): CSSRules<{}> {
```

Comment:
Isn't this effectively hard coding this styling?  I'm thinking the vertical positioning of the flippers might be something that designers might want to modify based on slide content.

Reply:
Yeah, but I believe this is the correct way we want devs to change styles. They can simply override with `jssStyleSheet`. We want the MSFT shape/style to cover the widest ground, and I think flippers in the middle covers this. Happy to make design updates either way.